# Message from the Global Planner to the local planner defining the desired waypoints

Header header
geometry_msgs/PoseStamped[] poses
nav_msgs/MapMetaData info
int8[] data